POS/Operation/03_EditBillList.md

```diff
@@ -3,181 +3,172 @@
+    > [Info] You have to enter a quantity higher than 0. The ![Minus](../../Assets/Icons/Minus.png "[Minus]") (Minus) button is only active if the quantity is higher than one. If you want to set the quantity to zero, remove the offer from the bill list, see [Delete an offer from the bill list](03_EditBillList.md#delete-an-offer-from-the-bill-list).
 
+3. Click the [SAVE] button in the bottom right corner to apply the quantity change.   
+    The detail view of the offer is closed. The bill list is displayed again. The selected quantity is displayed to the left of the offer.
 
 
-## Edit the quantity of a product
+## Check the stock level
 
-You can change the quantity of a product in your bill list by increasing or decreasing it.
+You can check the stock level respectively the availability of an offer for the current store or all other stores of your company.
 
 #### Prerequisites
 
 - Offers for POS are created, see [Manage offers for POS](../Integration/07_ManageOffers.md).
-- At least one product is added to the bill list, see [Select an offer](./02_SelectOffer.md).
+- At least one offer is added to the bill list, see [Select an offer](./02_SelectOffer.md#select-an-offer).
 
 #### Procedure
 
 *Venduo POS > Sales > Select store and pay desk > Tab CASHPOINT*
 
 ![Cashpoint](../../Assets/Screenshots/POS/Sales/Cashpoint/Product.png "[Cashpoint]")
 
-1. Click the product in the bill list whose quantity you want to change.   
-    The detail view of the product is displayed in the bill list.
+1. Click the offer in the bill list whose stock level you want to check.   
+    The detail view of the offer is displayed in the bill list.
 
-    ![Product Detail](../../Assets/Screenshots/POS/Sales/Cashpoint/BillList/ProductDetails.png "[Product Detail]")
+    ![Offer detail](../../Assets/Screenshots/POS/Sales/Cashpoint/BillList/ProductDetails.png "[Offer detail]")
 
-2. To the right of the *Quantity* field, click the ![Plus](../../Assets/Icons/Plus03.png "[Plus]") (Plus) button to increase the quantity or click the ![Minus](../../Assets/Icons/Minus.png "[Minus]") (Minus) button to decrease the quantity of the selected product. Alternatively, you can directly enter a number in the text field to the left of the buttons.
+2. Click the [SHOW STOCK LEVELS] button.   
+    The *Stock levels* window is displayed. 
 
-    > [Info] You have to enter a quantity higher than 0. The ![Minus](../../Assets/Icons/Minus.png "[Minus]") (Minus) button is only active if the quantity is higher than 1.
+    ![Stock levels](../../Assets/Screenshots/POS/Sales/Cashpoint/BillList/StockLevels.png "[Stock levels]")
 
-3. Click the [SAVE] button in the bottom right corner to apply the quantity change.   
-    The detail view of the product is closed. The bill list is displayed again. The selected quantity is displayed to the left of the product.
+3. Check the availability of the offer.
```

Comment:
availability is "tricky". Available stock in Warehousing bedeutet stock level - reservierungen. In POS ist vielleicht nicht so relevant, aber evtl. ein anderes Wort suchen? In stock? -> Check the number of units/items in stock for the desired offer.